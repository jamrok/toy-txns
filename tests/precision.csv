type,        client,  tx,  amount
deposit,     1,       1,   1.01
deposit,     2,       2,   2.023
deposit,     1,       3,   2.0456
withdrawal,  1,       4,   1.57891
withdrawal,  2,       5,   1.023456
