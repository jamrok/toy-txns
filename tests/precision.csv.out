client,available,held,total,locked
1,1.4767,0,1.4767,false
2,0.9995,0,0.9995,false
